src/teehr/queries/duckdb.py

```diff
@@ -188,6 +188,8 @@ def get_metrics(
             {tqu._remove_duplicates_mq_cte(mq)}
         )
         {tqu._nse_cte(mq)}
+        {tqu._annual_metrics_cte(mq)}
+        {tqu._spearman_ranks_cte(mq)}
```

Comment:
Just for my own knowledge, these are the "pre-calculated" metrics which are used `metrics` CTE below, is that correct?